ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -348,7 +353,7 @@ export default class SecurityTab extends PureComponent {
 
     return (
       <div
-        ref={this.settingsRefs[7]}
+        ref={this.settingsRefs[4]}
         className="settings-page__content-row"
         data-testid="advanced-setting-gas-fee-estimation"
```

Comment:
[out of scope] we'll want to update this `data-testid` value to reflect the related setting tab and setting